addons/web/static/src/core/utils/strings.js

```diff
@@ -2,32 +2,6 @@ import { isObject } from "./objects";
 
 export const nbsp = "\u00a0";
 
-/**
- * Escapes a string for HTML.
- *
- * @param {string | number} [str] the string to escape
- * @returns {string} an escaped string
- */
-export function escape(str) {
```

Comment:
Note: if you are looking for this function, use `htmlEscape` from owl instead, and then use `toString()` if you need to remove the markup class.